src/Symfony/Validator/Validator.php

```diff
@@ -28,7 +28,7 @@
  */
 class Validator implements ValidatorInterface
 {
-    public function __construct(private readonly SymfonyValidatorInterface $validator, private readonly ?ContainerInterface $container = null)
+    public function __construct(private readonly SymfonyValidatorInterface $validator, private readonly ContainerInterface $locator)
```

Comment:
It's a breaking change, we cannot do this since it was not deprecated beforehand.

Reply:
Even if the class is marked as final? 

I can revert the null value if needed. Changing name is ok?

Reply:
Yes! You should read https://symfony.com/doc/current/contributing/code/bc.html#working-on-symfony-code, we follow the same rules.
In this case, we are changing an argument type for a public method. Even if the class is marked as `final`, it is not allowed.
You can change the name though, it is allowed (but not recommended).